### ISSUE TITLE
Alt text button inside the image editor does not work on first load

### ISSUE THREAD

**Issue by ragnarok986** [DMKEBUSINESSGMBH/typo3-mkcontentai#17]
Hello,

We have an issue where the alt text generate button when we edit an existing element and an image field inside it, does not work on first open.
If we close the element or reload frame, the button then works as intended.

Same issue with newly created elements; once saved works as intended.
Any attempt after the first edit works as intended.

Console log also shows no js issue as much as can be seen.

TYPO3 version: 13.4.13
MKContentAI version: 13.0.10
Elements created with content blocks.

If any additional feedback needed, just let me know.

Best Regards

**Comment by hannesbochmann** [comment]
Thank you for your report. We will take a look at it, but I can't tell you when we will find the time.

**Comment by hannesbochmann** [comment]
This bug was fixed with the release of version 13.0.12
